<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelIsarmed">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>是否解锁</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>控制模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTargetAttitude">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标姿态</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRealAttitude">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>实时姿态</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelIsarmed">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Locked</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelMode">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stabilized</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelTargetRoll">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArm">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>解锁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisarm">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>锁定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModeStabilized">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>自稳模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModePosition">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>定姿模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModeForward">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>前向模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRoll">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>roll:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPitch">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYaw">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>410</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>yaw:</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelRealRoll">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTargetUpdate">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRoll">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPitch">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>370</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYaw">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>410</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSerial">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSerialOpen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PortBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSerialRefresh">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>92</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRoll_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>roll:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPitch_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYaw_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>yaw:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYaw_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>yaw:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPitch_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRoll_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>roll:</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelTargetPitch">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelTargetYaw">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelRealPitch">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="showLabelRealYaw">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
